relay-server/src/utils/metrics_rate_limits.rs

```diff
@@ -45,30 +37,34 @@ const PROFILE_TAG: &str = "has_profile";
 /// Additionally tracks whether the transactions also contained profiling information.
 ///
 /// Returns `None` if the metric was not extracted from transactions.
-fn count_metric_bucket(metric: BucketView<'_>, mode: ExtractionMode) -> Option<TransactionCount> {
+fn count_metric_bucket(metric: BucketView<'_>, mode: ExtractionMode) -> BucketSummary {
```

Comment:
Probably can get a better name now, something something summary?

Reply:
How about `BucketSummary::new`?

Reply:
Renamed to `summarize_bucket` to keep the diff small.